xblock/core.py

```diff
@@ -287,11 +297,7 @@ def needs_serialization(self):
         If all of the aside's data is empty or a default value, then the aside shouldn't
         be serialized as XML at all.
         """
-        return any([field.is_set_on(self) for field in self.fields.itervalues()])
-
-
-# Maintain backwards compatibility
-import xblock.exceptions
+        return any(field.is_set_on(self) for field in future.utils.itervalues(self.fields))
```

Comment:
I thought one of the advantages of future was that it didn't put stuff like "future.utils.itervalues" in your code? I can see it would be difficult to avoid this automatically.